src/Microsoft.SqlTools.ServiceLayer/DacFx/ExtractOperation.cs

```diff
@@ -30,7 +30,8 @@ public ExtractOperation(ExtractParams parameters, ConnectionInfo connInfo) : bas
         public override void Execute()
         {
             Version version = ParseVersion(this.Parameters.ApplicationVersion);
-            this.DacServices.Extract(this.Parameters.PackageFilePath, this.Parameters.DatabaseName, this.Parameters.ApplicationName, version, null, null, null, this.CancellationToken);
+            DacExtractOptions extractOptions = GetExtractOptions(this.Parameters.ExtractTarget);
+            this.DacServices.Extract(this.Parameters.PackageFilePath, this.Parameters.DatabaseName, this.Parameters.ApplicationName, version, null, null, extractOptions, this.CancellationToken);
```

Comment:
Let's take this opportunity to turn the magic constants into named parameters.

Reply:
Done.